rebar.lock

```diff
@@ -14,5 +14,13 @@
  {<<"jhn_stdlib">>, <<"F27A0F71F79C54AF79E8E7C1DFC5BE15969F8399ABC0EBDF5B201EB442FCDFD7">>},
  {<<"quickrand">>, <<"665C49881808DA68A5C58DDEDA16EA60B16BE54DBB719B029EEA5835FD12DAE9">>},
  {<<"ranch">>, <<"6B1FAB51B49196860B733A49C07604465A47BDB78AA10C1C16A3D199F7F8C881">>},
- {<<"uuid">>, <<"02C713FAC0E73E5F366CBA541216F1E956E2F00C91C7530D4B21DEC7D3ADA164">>}]}
+ {<<"uuid">>, <<"02C713FAC0E73E5F366CBA541216F1E956E2F00C91C7530D4B21DEC7D3ADA164">>}]},
+{pkg_hash_ext,[
+ {<<"cowboy">>, <<"4643E4FBA74AC96D4D152C75803DE6FAD0B3FA5DF354C71AFDD6CBEEB15FAC8A">>},
+ {<<"cowlib">>, <<"E4175DC240A70D996156160891E1C62238EDE1729E45740BDD38064DAD476170">>},
+ {<<"erlydtl">>, <<"A366C4CD32E8B54D7D0825AA51EF6A12C35D6D558BEFFC276275AD335B1FDBED">>},
+ {<<"jhn_stdlib">>, <<"15F38D1EB9F0DD15FFB070633E3D0263072A439BC0A9A7F79ECDCB6575D3CD04">>},
+ {<<"quickrand">>, <<"0344D02B40BC44B3E8E8A3DBD89A3B29E5690BC44811149EE704AC3A695A29E3">>},
+ {<<"ranch">>, <<"451D8527787DF716D99DC36162FCA05934915DB0B6141BBDAC2EA8D3C7AFC7D7">>},
+ {<<"uuid">>, <<"CE1C1ED1D786BA98F93204BE66F93A797504993954141122FCC5FE59B5331367">>}]}
```

Comment:
Why is there two entries of uuid here :S

Reply:
hmm.. not really sure. Will see if I can solve it.